modules/4337/package.json

```diff
@@ -76,6 +78,6 @@
     }
   },
   "dependencies": {
-    "@safe-global/safe-contracts": "^1.4.1"
+    "@safe-global/safe-contracts": "^1.4.1-build.0"
```

Comment:
Out of curiosity, what is the difference?

Reply:
1.4.1 pointed to a non-existing js file as the entry file. It didn't affect our build, but the autocomplete suggested this version as the latest one in the passkeys package.json, and when you try to use 1.4.1 and 1.4.1-build.0 together, npm will complain :shrug: 

A full overview of the changes: https://github.com/safe-global/safe-smart-account/compare/v1.4.1...v1.4.1-build.0

Reply:
Thanks!